flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/connector/print/table/PrintTableSinkFactory.java

```diff
@@ -126,6 +128,9 @@ public DynamicTableSink copy() {
         public String asSummaryString() {
             return "Print to " + (stdErr ? "System.err" : "System.out");
         }
+
+        @Override
+        public void applyStaticPartition(Map<String, String> partition) {}
```

Comment:
I think it is better to print to StaticPartition, we can append it to printIdentifier.

Reply:
Done for print StaticPartition, and added some tests in `PrintTableSinkFactoryTest.java` & `BlackHoleSinkFactoryTest.java`.